### ISSUE TITLE
UI drawer overlap issue

### ISSUE THREAD

**Issue by womeier** [nextcloud/android#11337]
- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. open app

### Expected behaviour

![danach](https://user-images.githubusercontent.com/55190123/215866827-11ae521f-8e6f-45c7-b2b2-49d247cbc5db.png)

### Actual behaviour

![davor](https://user-images.githubusercontent.com/55190123/215866467-97d5b5fb-f097-4cac-925d-d6ebb6c9a275.png)

### Android version

11

### Device brand and model

Pixel XL (virtual device)

### Stock or custom OS?

Stock

### Nextcloud android app version

master branch as of 31.1.23 (051513d)

### Nextcloud server version

not relevant

### Using a reverse proxy?

No

**Comment by womeier** [comment]
I'd like to fix this, but I wasn't able to find the correct color (for the dark theme)

It seems, the drawer background depends on a theme/style and is not "@color/appbar" on my phone.

**Comment by AlvaroBrey** [comment]
As I commented in #11336:

> This should probably be fixed by setting a padding or margin in some of the overlapping elements, rather than a background color; I haven't had time to look at it unfortunately

**Comment by satvik2131** [comment]
What could be a better approach  , to change the background color , or to make the quota bar at the bottom and when we scroll up it will not be shown , please suggest ? @ZetaTom

**Comment by AndyScherzinger** [comment]
Hi at @satvik2131 - see https://github.com/nextcloud/android/issues/11337#issuecomment-1427714404 either via a background color or finding a way for padding/margin to prevent scrolling behind the quota.

**Comment by satvik2131** [comment]
hey , I know this must not be the right place to ask , but I want to discuss some doubts , is there any discord channel or any communication medium ?? 

Question: the question is like if we scroll should the quota bar should hide or have to be shown ??

**Comment by ZetaTom** [comment]
> hey , I know this must not be the right place to ask , but I want to discuss some doubts , is there any discord channel or any communication medium ??
> 
> Question: the question is like if we scroll should the quota bar should hide or have to be shown ??

That is a good question. I think we should leave that decision up to @nextcloud/designers.

---

To restate the issue, as I understand it:
Currently, the quota display (e.g. *2.9 MB of 5.0 GB used*) overlaps with the navigation bar on certain devices. To fix this, we have two options
1. we either move the quota display to the bottom, such that it always appears at the bottom of the navigation drawer with an opaque background, or
2. add it a list entry to the bottom of the navigation items (*All files*, *Favourites*, etc.).

In the first case, navigation items would scroll above the quota, which would always be visible. In the second case, the drawer would be filled to the bottom with as many navigation items, as would fit on screen and the quota display would be added on as an extra item. In this case, on certain devices, users would have to scroll down to see see the quota.

**Comment by szaimen** [comment]
> > Question: the question is like if we scroll should the quota bar should hide or have to be shown ??
> 
> That is a good question. I think we should leave that decision up to @nextcloud/designers.

Yes, I think the quota bar should be part of the list and thus scroll.

**Comment by jancborchardt** [comment]
> we either move the quota display to the bottom, such that it always appears at the bottom of the navigation drawer with an opaque background

This is how it should be actually (fyi @szaimen @ZetaTom , not as part of the list).
- Sticky on the bottom of the drawer, like in Files
- Opaque background of course

So adding an opaque background here could be an easy fix? :)

**Comment by bhavesh100** [comment]
Here I have created an opaque background.

You can see what it looks like here.
Would you like to give any suggestions?

https://github.com/nextcloud/android/assets/90559083/fd59a82a-78e4-4b17-a9bd-e5f3fb2ae5f6

**Comment by AndyScherzinger** [comment]
@bhavesh100 looks great 👍 Did you also implement it for dark theme?

Looking forward to your PR 🎉

**Comment by bhavesh100** [comment]
@AndyScherzinger Thank you, for your review ✨

I am using the background color which is already defined and used in the side bar navigation drawer.

```kt
android:background="@color/appbar"
```

It doesn't need to be implemented for both Theme.

<img src="https://github.com/nextcloud/android/assets/90559083/dc1de2ce-2339-4308-a192-dfb394e9e226" width="300" height="300">
<img src="https://github.com/nextcloud/android/assets/90559083/e6843949-16e9-4446-ad9a-83d65ec7451c" width="300" height="300">
<p>

But I am facing this problem...the last element is hidden behind it.

I know how to resolve it.

This is easily solved by increasing the space below the community element.

</p>

<img src="https://github.com/nextcloud/android/assets/90559083/91915ddc-8775-4ca4-b837-211507e1aa40" width="400" height="400">

<p>

After resolving i will create a pull request.

</p>
